contracts/Vault.vy

```diff
@@ -222,7 +222,7 @@ totalDebt: public(uint256)  # Amount of tokens that all strategies have borrowed
 lastReport: public(uint256)  # block.timestamp of last report
 activation: public(uint256)  # block.timestamp of contract deployment
 lockedProfit: public(uint256) # how much profit is locked and cant be withdrawn
-lockedProfitDegration: public(uint256) # rate per block of degration. DEGREDATION_COEFFICIENT is 100% per block
+lockedProfitegradation: public(uint256) # rate per block of degradation. DEGRADATION_COEFFICIENT is 100% per block
```

Comment:
forgot a `d`

Reply:
Good catch, not sure how my find-replace managed to do that lol